.storybook/docsPageConfig.jsx

```diff
@@ -0,0 +1,25 @@
+import {
+  ArgsTable,
+  Description,
+  Primary,
+  PRIMARY_STORY,
+  Stories,
+  Subtitle,
+  Title,
+} from '@storybook/addon-docs';
+
+import { GuidelineLink } from '@config/GuidelineLink';
+
+export const configureDocsPage = (componentName) => {
+  return () => (
+    <>
+      <Title />
+      <Subtitle />
+      <Description />
+      {componentName && <GuidelineLink path={componentName} />}
+      <Primary />
+      <ArgsTable story={PRIMARY_STORY} />
+      <Stories />
+    </>
+  );
+};
```

Comment:
This is a noteworthy file; I extracted the common ordering of items for the **Docs** tab of each component into a reusable piece. This was previously specified implicitly in each story using MDX, but now gets added in the `parameters.docs.page` configuration of the default export for each component.